lib/lita/handlers/jira.rb

```diff
@@ -105,9 +112,18 @@ def myissues(response)
 
         response.reply(format_issues(issues))
       end
+
+      def ambient(response)
```

Comment:
RuboCop is complaining about cyclomatic complexity. Mind providing some advice on best practices here, @billkauf?